### PR TITLE
Add more logging to holiday-stop processing lambda

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

File: handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/Handler.scala [hunk 1]
```diff
@@ -2,28 +2,30 @@ package com.gu.holidaystopprocessor
...
       case Right(config) =>
         val processResult = HolidayStopProcess(config)
 
+        logger.log(s"${processResult.holidayStopsToApply.size} holiday stops to apply:")
+        processResult.holidayStopsToApply.foreach(stop => logger.log(stop.toString))
+
         processResult.holidayStopResults foreach {
-          case Left(failure) => logger.error(failure.reason)
-          case Right(response) => logger.info(response)
+          case Left(failure) => logger.log(failure.reason)
```
Comment (mario-galic) [inline handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/Handler.scala]: Are we losing log levels here? Why not extend `LazyLogging` from `scala-logging`?

Comment (kelvin-chappell) [reply handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/Handler.scala]: Reverted to how it was now, but with console logger configured.

**Comment by kelvin-chappell** [comment]
I'll have a go at the for comprehension approach.

**Comment by kelvin-chappell** [comment]
@mario-galic I think the difficulty in the for-comprehension is that I want to export any amendments that succeed and log failures.  But that's difficult to do neatly in a for-comprehension.  Can pair on this later if you can see a solution.  But will do in a separate PR.

**Pull request merged**